lib/model/message.dart

```diff
@@ -502,10 +552,18 @@ class MessageStoreImpl extends HasChannelStore with MessageStore, _OutboxMessage
   }
 
   void handleMessageEvent(MessageEvent event) {
+    final message = event.message;
+
     // If the message is one we already know about (from a fetch),
     // clobber it with the one from the event system.
     // See [fetchedMessages] for reasoning.
```

Comment:
(Is this existing comment meant to refer to `reconcileMessages`? I think `fetchedMessages` was a previous name for that.)